### DISCUSSION TITLE
IINA 1.3.3

### DISCUSSION THREAD

**Discussion by lhc70000** [iina/iina#4536] [Announcements]
## New
- Added an option to start iina-cli in music mode [#3651](https://github.com/iina/iina/issues/3651)
- IINA is now associated with .gif files [#3679](https://github.com/iina/iina/issues/3679)

## Bug fixes
- FFmpeg 6.0 has issues when hardware decoding VP9 format on Intel Macs. Therefore, you might experience this freezing when trying to play YouTube videos in IINA 1.3.2. We have temporarily disabled hardware decoding for VP9 on Intel Macs [#4486](https://github.com/iina/iina/issues/4486)
- Fixed missing localization strings that appears in v1.3.2 [#4481](https://github.com/iina/iina/issues/4481)
- Fixed a bug where audio delay cannot be set to negative values [#4482](https://github.com/iina/iina/issues/4482)
- Fixed an issue where stop command doesn't close the mini player [#4398](https://github.com/iina/iina/pull/4398)
- Fixed failure to write '#' to keybinding file [#4271](https://github.com/iina/iina/pull/4271)
- Fixed filter parameters appears randomly [#4262](https://github.com/iina/iina/issues/4262)

## Improvements
- When loading subtitles, the open panel always starts in the video's directory [#4382](https://github.com/iina/iina/issues/4382)
- IINA can correctly handle TAB and ESC as keybindings [#4465](https://github.com/iina/iina/issues/4465)
- Pass folder to mpv directly to make mpv-shuffle working [#4434](https://github.com/iina/iina/issues/4434)
- Add a tick mark to A-B loop in the menu when A-B loop is enabled [#4419](https://github.com/iina/iina/pull/4419)

---

**SHA256**
```
4b3f6c4bed3bb77dbe29c12bf6d5d0959284afb01c7b59a35fd71a3a27088991
```

<hr /><em>This discussion was created from the release <a href='https://github.com/iina/iina/releases/tag/v1.3.3'>IINA 1.3.3</a>.</em>

**Comment by valynor** [comment]
Release was 10h ago but is still not showing even when you manually check for updates in IINA 1.3.2.

**Comment by dougless12** [comment]
is there an option to remove/unremove pictures from a playlist or so it doesnt add pics when opening a folder?

**Reply by justinmayer** [reply]
This seems to be a bug in version 1.3.3. I reverted to 1.3.2 and filed a report: #4548

**Comment by Belogvardeyets** [comment]
Hello. Great player! Thanks a lot. It's a shame it's not available for Windows.
